src/test/java/redis/clients/jedis/tests/SSLJedisSentinelPoolTest.java

```diff
@@ -63,8 +57,7 @@ public void sentinelWithSslConnectsToRedisWithSsl() {
         .hostAndPortMapper(SSL_PORT_MAPPER).build();
     DefaultJedisClientConfig sentinelConfig = DefaultJedisClientConfig.builder()
         .clientName("sentinel-client").ssl(true).hostAndPortMapper(SSL_PORT_MAPPER).build();
-    try (JedisSentinelPool pool = new JedisSentinelPool(MASTER_NAME, sentinels, POOL_CONFIG,
-        masterConfig, sentinelConfig)) {
+    try (JedisSentinelPool pool = new JedisSentinelPool(MASTER_NAME, sentinels, masterConfig, sentinelConfig)) {
```

Comment:
@sazzad16 do we still have other tests that cover the original constructor?

Reply:
@gkorland added those back.